dev/com.ibm.ws.javaee.ddmodel/resources/com/ibm/ws/javaee/internal/ddmodel/resources/DDModelMessages.nlsprops

```diff
@@ -70,21 +83,94 @@ invalid.href.prefix=CWWKC2260E: The href attribute value of the {2} element does
+
+# REPLACED BY:
+# -----------------------------------------------------------------------
+# PARAMETERS:
+#    {0} is the path to a deployment descriptor, including paths to parent archives
+#    {1} is the number of a line within the deployment descriptor
+#    {2} is a version attribute value within the deployment descriptor
+# -----------------------------------------------------------------------
+unsupported.descriptor.version=CWWKC2261E: The deployment descriptor {0}, at line {1}, specifies unsupported version {2}.
+unsupported.descriptor.version.explanation=The version which was specified in the deployment descriptor is not valid for that type of deployment descriptor.
+unsupported.descriptor.version.useraction=Correct the version specified in the deployment descriptor.
 
-invalid.deployment.descriptor.namespace=CWWKC2262E: The server is unable to process the {3} version and the {2} namespace in the {0} deployment descriptor on line {1}.
-invalid.deployment.descriptor.namespace.explanation=Either the specified version and namespace are not valid, or the server configuration does not have a feature that can process the version and namespace.
-invalid.deployment.descriptor.namespace.useraction=Verify and correct the version or namespace in the root element of the deployment descriptor, or verify and correct the features in the server configuration.
+# REMOVED:
+# invalid.deployment.descriptor.namespace=CWWKC2262E: The server is unable to process the {3} version and the {2} namespace in the {0} deployment descriptor on line {1}.
+# invalid.deployment.descriptor.namespace.explanation=Either the specified version and namespace are not valid, or the server configuration does not have a feature that can process the version and namespace.
+# invalid.deployment.descriptor.namespace.useraction=Verify and correct the version or namespace in the root element of the deployment descriptor, or verify and correct the features in the server configuration.
 
-invalid.deployment.descriptor.version=CWWKC2263E: The version {2} attribute specified in the {0} deployment descriptor on line {1} is not valid.
-invalid.deployment.descriptor.version.explanation=An invalid version was specified in the root element of the deployment descriptor.
-invalid.deployment.descriptor.version.useraction=Verify and correct the version number in the root element of the deployment descriptor.
+# REPLACED BY:
+# -----------------------------------------------------------------------
+# PARAMETERS:
+#    {0} is the path to a deployment descriptor, including paths to parent archives
+#    {1} is the number of a line within the deployment descriptor
+#    {2} is a namespace value within the deployment descriptor
+# -----------------------------------------------------------------------
+unsupported.descriptor.namespace=CWWKC2262E: The deployment descriptor {0}, at line {1}, specifies unsupported namespace {2}.
```

Comment:
Change "deployment descriptor {0}" to "{0} deployment descriptor"

Reply:
Updated.